bin/import_training_speech.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+from __future__ import absolute_import, division, print_function
+
+# Make sure we can import stuff from util/
+# This script needs to be run from the root of the DeepSpeech repository
+import os
+import re
+import sys
+
+
+sys.path.insert(1, os.path.join(sys.path[0], '..'))
+
+import csv
+import zipfile
+import progressbar
+import requests
+
+from os import path
+from util.text import validate_label
+
+FIELDNAMES = ['wav_filename', 'wav_filesize', 'transcript']
+MAX_SECS = 10
+ARCHIVE_DIR_NAME = 'training_speech'
+ARCHIVE_NAME = '2018-09-28_fr_FR.zip'
+ARCHIVE_URL = 'https://s3.eu-west-3.amazonaws.com/audiocorp/releases/' + ARCHIVE_NAME
+
+SIMPLE_BAR = ['Progress ', progressbar.Bar(), ' ', progressbar.Percentage(), ' completed']
+
+
+def _download_and_preprocess_data(target_dir):
+    # Making path absolute
+    target_dir = path.abspath(target_dir)
+    # Conditionally download data
+    archive_path = _maybe_download(ARCHIVE_NAME, target_dir, ARCHIVE_URL)
+    # Conditionally extract common voice data
+    _maybe_extract(target_dir, ARCHIVE_DIR_NAME, archive_path)
+    # Conditionally convert common voice CSV files and mp3 data to DeepSpeech CSVs and wav
+    _maybe_convert_sets(target_dir, ARCHIVE_DIR_NAME)
+
+def _maybe_download(archive_name, target_dir, archive_url):
```

Comment:
@nicolaspanel Sorry, I just landed something to take care of that, you can remove that code :)